src/PublicAssets.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Wrapper class for the enqueue function because we can't autoload functions.
+ */
+final class PublicAssets {
+	/**
+	 * Load styling and behaviour on website pages.
+	 */
+	public static function enqueue(): void {
+		$theme_dir = get_template_directory_uri();
```

Comment:
For now I preserved this to not change the logic too much while moving the code, but it can probably also use `Loader:enqueue_versioned_style`, which gets this internally.